### PR TITLE
modifying migration section with new terminology

### PR DESCRIPTION
**Pull request by emuus**
A certain number of screenshots still need to be updated, as does the getting started page.

### CODE REVIEW

File: end-user/en/source/pages/migration/scan-to-appliance.rst [hunk 1]
```diff
@@ -92,14 +92,7 @@ For a list of supported OSes for Migration, see the table in :ref:`uforge-suppor
 
 Major OS versions, for example upgrading from CentOS 5.0 to CentOS 6.0 is not supported automatically, though as we have the complete list of operating system packages from the scan, a new appliance template can be constructed with the new operating system version.  
 
-This process can further be automated by using the command-line tool hammr (see `hammr.io <http://www.hammr.io>`_).  This tool allows you to create identical machine images from a single configuration file (in JSON).  The procedure would be to:
```
Comment (cinlloc) [inline end-user/en/source/pages/migration/scan-to-appliance.rst]: I'm not sure to see how this change is linked to the current renaming. If it's not linked and you prefer not to do it in another commit, please tell about it in the commit message.

Comment (emuus) [reply end-user/en/source/pages/migration/scan-to-appliance.rst]: Sorry yes, I spotted this while doing the terminology changes but it is linked to the changes to hammr migration tool. Please let me know if I should remove from this commit and add to the hammr PR

File: end-user/en/source/pages/migration/blackbox-migration.rst [hunk 2]
```diff
@@ -5,12 +5,12 @@
 Blackbox Migration Process
 --------------------------
 
-The goal of black box migration (either using ``Scans`` or ``Migrations``) is to reproduce a near identical copy of the currently running workload.  However, there will always be small differences between the two workloads after migration is complete, notably some services are disabled or enabled depending on the target machine image being created (refer to :ref:`service-state`). For more information on the difference after migration, refer to :ref:`source-target-diffs`. 
+The goal of black box migration (either using ``Re-platform`` or ``Lift & Shift``) is to reproduce a near identical copy of the currently running workload.  However, there will always be small differences between the two workloads after migration is complete, notably some services are disabled or enabled depending on the target machine image being created (refer to :ref:`service-state`). For more information on the differences after migration, refer to :ref:`source-target-diffs`.
```
Comment (segalaj) [inline end-user/en/source/pages/migration/blackbox-migration.rst]: we use ``blackbox`` in one word in the rest of the doc

File: end-user/en/source/pages/migration/overview.rst [hunk 3]
```diff
@@ -7,12 +7,12 @@ Migrating Live Workloads
 
 UForge AppCenter offers the capability to migrate a live system using two different methods:
 
-* using ``Scans``. In this case there are three steps to migration: scan, generate and publish. Each step is done seperately by the user. When using scan to migrate, you can modify the appliance prior to the final migration.
-* using ``Migration``. This automated process is only possible for Linux and certain target platforms.
+* using ``Re-platform``. This was previously refered to as ``Scan``. In this case there are three steps to migration: scan, generate and publish. Each step is done seperately by the user. When using scan to migrate, you can modify the appliance prior to the final migration.
+* using ``Lift & Shift``. This was previously refered to as ``Migrate``. This automated process is only possible for Linux and certain target platforms.
```
Comment (cinlloc) [inline end-user/en/source/pages/migration/overview.rst]: As "Migrations" concept was introduced in FP, do we have to give that indication?

Comment (emuus) [reply end-user/en/source/pages/migration/overview.rst]: Good question. @jfknoepfli what do you think? do any clients move from one fp to another fp? if not then Clement is right and there is no need to mention "migrate" as not in 3.8

File: end-user/en/source/pages/migration/blackbox-migration.rst (hunk 2 above)
Comment (emuus) [reply end-user/en/source/pages/migration/blackbox-migration.rst]: I also noticed this inconsistency. Both are used in our doc. I've checked again and it is best to use "black box". I will try to fix for this fp but if not it can be raised done as an improvement on future release.

File: end-user/en/source/pages/migration/overview.rst (hunk 3 above)
Comment (cinlloc) [reply end-user/en/source/pages/migration/overview.rst]: Discussed with @jfknoepfli: he would like to remove both "This was previously refered to as ``Scan``" and "This was previously refered to as ``Migrate``"

File: end-user/en/source/pages/migration/migration-details.rst [hunk 4]
```diff
@@ -11,7 +11,7 @@ The entire migration process using ``Re-platform`` (previously referred to as ''
 	2. Report the scan results to UForge AppCenter, where the platform analyzes the report. The scan differentiates between known packages and extra files. 
 	3. The results are used to build an archive. The extra files are only included in the archive if the user is performing a scan with overlay. The archive is sent back to the platform.
 	4. The archive is uncompressed, and the information is stored in the database as a ``Scan``.
-	5. The scan can be used to generate machine images (also known as blackbox migration) or imported to create a new appliance template (also known as whitebox migration). The generated machine image can then be published to the target environment and instances can be provisioned.
+	5. The scan can be used to generate machine images (also known as black box migration) or imported to create a new appliance template (also known as whitebox migration). The generated machine image can then be published to the target environment and instances can be provisioned.
```
Comment (cinlloc) [inline end-user/en/source/pages/migration/migration-details.rst]: Should not be "white box"?
